packages/eslint-plugin/src/lib/configs/all.ts

```diff
@@ -22,10 +22,28 @@ export const barrelModulesOnly: TSESLint.FlatConfig.Config = {
   },
 };
 
-export const all: TSESLint.FlatConfig.Config = {
-  ...commonConfig,
+export const validateSheriffConfig: TSESLint.FlatConfig.Config = {
+  files: ['sheriff.config.ts'],
```

Comment:
This config is in conflict with my current recommendation for user config as follows:

```
module.exports = tseslint.config({
  files: ['**/*.ts'],
  extends: [...sheriff.configs.all],
});
```

At the moment, this is what is stopping me from submitting this pr.

Reply:
No longer an issue. I've chosen to use file exclusion in the code rather than inside eslint config.